satellite/satellitedb/database.go

```diff
@@ -59,10 +60,10 @@ func (db *DB) StatDB() statdb.DB {
 	return &statDB{db: db.db}
 }
 
-// // OverlayCacheDB is a getter for OverlayCacheDB repository
-// func (db *DB) OverlayCacheDB() overlay.DB {
-// 	return &overlayCacheDB{db: db.db}
-// }
+// OverlayCache is a getter for overlay cache repository
+func (db *DB) OverlayCache() storage.KeyValueStore {
```

Comment:
We want a better interface here than `storage.KeyValueStore`, but it's fine for now.